src/main.js

```diff
@@ -22,15 +21,5 @@ app.use(store);
 
 app.mount('#app');
 
-////////////////////////
-// Development Config //
-////////////////////////
-
-if (!import.meta.env.PROD || window.Cypress) {
-  // Expose app for debugging/testing
-  window.cuttle = {
-    app,
-  };
-  // Connect the devtools -- non-prod only
-  devtools.connect(null, 8098);
-}
+// Add Cuttle window object
+initCuttleGlobals(app);
```

Comment:
Moved all globals logic to a singular function. This also means we could mock it in Vittest if we really want extra coverage eventually.

Reply:
NIce cleanup